python/tvm/script/tir/__init__.pyi

```diff
@@ -210,9 +210,14 @@ def alloc_buffer(
 """
 special_stmt - Reads/Writes
 """
-
+@overload
 def reads(read_regions: Union[BufferSlice, List[BufferSlice]]) -> None: ...
+@overload
+def reads(read_regions: BufferSlice, *other_regions: BufferSlice) -> None: ...
```

Comment:
From the user-facing side, if we have two overloading versions I don't think `other_regions` is necessary. We can instead do something like
```
@overload
def reads(read_regions: List[BufferSlice]) -> None: ...
@overload
def reads(*read_regions: BufferSlice) -> None: ...
```

Reply:
done